src/components/Dialogs/Dialog/Dialog.scss

```diff
@@ -1,13 +1,26 @@
 .Dialog {
-    width: 400px;
 
-    &__content {
+    &__header {
         padding: var(--space-200);
     }
 
+    &__trigger {
+         all: unset;
+         cursor: pointer;
+    }
+
+    &__dialog {
+        width: 400px;
```

Comment:
this can be at the root of .Dialog, the div that wraps the component is not needed